Mlem/Models/Trackers/Saved Account Tracker.swift

```diff
@@ -29,12 +29,18 @@ class SavedAccountTracker: ObservableObject {
     }
     
     func addAccount(account: SavedAccount) {
+        // prevent dupes
+        guard !savedAccounts.contains(account) else {
+            return
```

Comment:
actually, do we need a message returned here?

Reply:
I don't think so--it just quietly succeeds, user doesn't need to know they logged into an account they already had lol